src/NzbDrone.Core/Indexers/Definitions/Anilibria.cs

```diff
@@ -0,0 +1,329 @@
+using System;
+using System.Collections.Generic;
+using System.Collections.Specialized;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using AngleSharp.Html.Parser;
+using FluentValidation;
+using Newtonsoft.Json;
+using NLog;
+using NzbDrone.Common.Http;
+using NzbDrone.Core.Annotations;
+using NzbDrone.Core.Configuration;
+using NzbDrone.Core.Indexers.Exceptions;
+using NzbDrone.Core.IndexerSearch.Definitions;
+using NzbDrone.Core.Messaging.Events;
+using NzbDrone.Core.Parser;
+using NzbDrone.Core.Parser.Model;
+using NzbDrone.Core.Validation;
+
+namespace NzbDrone.Core.Indexers.Definitions
+{
+    public class Anilibria : TorrentIndexerBase<AnilibriaSettings>
+    {
+        public override string Name => "Anilibria";
+        public override string[] IndexerUrls => new string[] { "https://anilibria.tv/" };
+        public override string Description => "Так звучит Аниме!";
```

Comment:
Can we get this in English as well?

Reply:
I'm not sure what should be in description. Right now I've just copied slogan from main page of anilibria

Reply:
description of the site; ref for examples https://github.com/Prowlarr/Prowlarr/commit/0ddc530dd4b374980436eed15ed4fb0ac88b80ea

seems it is a anime tracker with russian dubs?

Reply:
Yep

Reply:
@bakerboy448 Changed description